README.md

```diff
@@ -72,7 +72,14 @@ Download the latest binary executable for your operating system:
   dnf install tektoncd-cli
   ```
 
-If you have [go](https://golang.org/) installed, `GO111MODULE="on" go get github.com/tektoncd/cli@v0.5.0` is all you need!
+* Source install, If you have [go](https://golang.org/) installed and you want
+  to compile the CLI from source you can check-out the [GitHUB repository](https://github.com/tektoncd/cli) and do a :
```

Comment:
1. remove `-` from `checkout-out` 
2. Change `GitHUB` to `GitHub` 
3. Change `do a` to `and run the following command:`